js-exercises/exercises.js

```diff
@@ -0,0 +1,182 @@
+/*
+  Reverse a string
+  1. Split the string into an array
+  2. Reverse the array
+  3. Join the array back into a string
+*/
+
+const reverseString = (str) => {
+  return str.split("").reverse().join("");
```

Comment:
Try to implement without using reverse. For example, you can use reduce

Reply:
Have updated to use reduce